src/test/java/utils/VacancySQLDataProvider.java

```diff
@@ -0,0 +1,41 @@
+package utils;
+
+import data.EmployeeSQLData;
+import data.JobVacancySQLData;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VacancySQLDataProvider extends SQLDataProvider{
```

Comment:
no need to extend from SQLDataProvider

Reply:
SQLDataProvider includes URL, USER and PASSWORD that are used by both providers